evalml/tests/data_checks_tests/test_data_check_action.py

```diff
@@ -5,7 +5,7 @@
 
 @pytest.fixture
 def dummy_data_check_name():
```

Comment:
nit: If you use this fixture in multiple files, would it be beneficial to move to the `conftest.py` file?